src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/listeners/PrisonerDomainEventsListener.kt

```diff
@@ -40,7 +60,6 @@ class PrisonerDomainEventsListener(
     val prisonerId: String
   )
 
-  data class SQSMessage(val Message: String, val MessageId: String)
-
-  private inline fun <reified T> String.fromJson(): T = objectMapper.readValue(this, T::class.java)
+  @JsonNaming(value = PropertyNamingStrategies.UpperCamelCaseStrategy::class)
+  data class SQSMessage(val Type: String, val Message: String, val MessageId: String)
```

Comment:
Need UpperCamelCaseStrategy to make jackson render Type as "Type" rather than "type"